### PR TITLE
Boris/amazon bid adapter

### PR DESCRIPTION
**Pull request by shlomisut**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

### CODE REVIEW

**Comment by ghost** [comment]
Build Failed - please check http://jenkins.thetimmedia.site:8080/job/prebid_pull_request/30/

File: modules/a9BidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    params=params.bids[0].params;
+    var sizes=params.sizes;
+    var serverDomain =params.serverDomain;
+    var slotId=params.slotId;
+    var slotName=params.slotName;
+    var pubID=params.pubID;
+    var apstagSlots = initializeAps(sizes,slotId,slotName,pubID);
+    apstag.fetchBids({
+      slots: apstagSlots,
+      timeout: 2e3
```
Comment (shlomisut) [inline modules/a9BidAdapter.js]: ha ?

**Comment by ghost** [comment]
Build Passed

**Review by borisjaskerovich — APPROVED** [review]

**Comment by ghost** [comment]
Build Passed

**Comment by ghost** [comment]
Build Passed

**Pull request merged**